graphql/task_resolver.go

```diff
@@ -578,6 +579,39 @@ func (r *taskResolver) TaskLogs(ctx context.Context, obj *restModel.APITask) (*T
 	return &TaskLogs{TaskID: utility.FromStringPtr(obj.Id), Execution: obj.Execution}, nil
 }
 
+// TaskOwnerTeam is the resolver for the taskOwnerTeam field.
+func (r *taskResolver) TaskOwnerTeam(ctx context.Context, obj *restModel.APITask) (*TaskOwnerTeam, error) {
+	fwsBaseURL := evergreen.GetEnvironment().Settings().FWS.URL
+	if fwsBaseURL == "" {
+		return nil, InternalServerError.Send(ctx, "Foliage Web Services URL not set")
+	}
+	httpClient := utility.GetHTTPClient()
+	defer utility.PutHTTPClient(httpClient)
+
+	cfg := fws.NewConfiguration()
+	cfg.HTTPClient = httpClient
+	cfg.Servers = fws.ServerConfigurations{
+		fws.ServerConfiguration{
+			Description: "Foliage Web Services",
+			URL:         fwsBaseURL,
+		},
+	}
+	cfg.UserAgent = "evergreen"
+
+	client := fws.NewAPIClient(cfg)
+	req := client.OwnerAPI.ByFoliageLogicApiOwnerByFoliageLogicTaskIdGet(ctx, *obj.Id)
+	results, _, err := req.Execute()
```

Comment:
The HTTP package requires that we close the response body returned in the 2nd parameter or else there will be a resource leak.